### DISCUSSION TITLE
Quick Access to favorite tags

### DISCUSSION THREAD

**Discussion by anandio-ml** [GitJournal/GitJournal#704] [Feature Requests]
## Description

**What you'd like to happen:**

have a quicker access to select tags. Maybe on the start screen or even as widget.

**Alternatives you've considered:**   

The 'problem' is actually more generic. I want to access currently important files much more quickly. Any solutions welcome.
What's important is of course up to the user ;) and may change any time.

Also seen #115 , but this is not to the point, because a user had to mark every single note manually inside of GitJournal.
Tag-Pinning would be much more powerful. Or maybe folder-pinning.

**Comment by gxingh** [comment]
I think #115 also makes a lot of sense if you want to pin 'todo' or 'inbox' notes to be seen at the top. 
Tag pinning could also be useful, like if you create todos using the #todo tag all over your workspace.

**Comment by anandio-ml** [comment]
Of course it makes sense! but it is a different use case. I am thinking more of topics you are currently working on than single items like todo lists. And to be able to pin documents from outside of your app by applying a certain label to it (like #pinned or #hot) or whatever.
